frontend/jest.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  testPathIgnorePatterns: ["<rootDir>/node_modules/"],
+  collectCoverage: true,
+  collectCoverageFrom: [
+    "src/**/*.js", // Include all JavaScript files in the src directory and its subdirectories
```

Comment:
How many files does this glob match? Aren't most of the files in the project .ts or .tsx?

Reply:
good point